src/Altinn.Apps/AppTemplates/AspNet/App.IntegrationTests/Data/apps/tdd/sirius/config/process/process.bpmn

```diff
@@ -0,0 +1,63 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<bpmn2:definitions id="Altinn_Data_Confirmation_Process_Definition"
+  xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+  xmlns:bpmn2="http://www.omg.org/spec/BPMN/20100524/MODEL"
+  xmlns:bpmndi="http://www.omg.org/spec/BPMN/20100524/DI"
+  xmlns:dc="http://www.omg.org/spec/DD/20100524/DC"
+  xmlns:di="http://www.omg.org/spec/DD/20100524/DI"
+  xmlns:altinn="http://altinn.no"
+  xsi:schemaLocation="http://www.omg.org/spec/BPMN/20100524/MODEL BPMN20.xsd"
+  targetNamespace="http://bpmn.io/schema/bpmn" >
+  <bpmn2:process id="Altinn_Data_Confirmation_Process" isExecutable="false">
+    <bpmn2:startEvent id="StartEvent_1">
+      <bpmn2:outgoing>SequenceFlow_1</bpmn2:outgoing>
+    </bpmn2:startEvent>
+    <bpmn2:task id="Task_1" name="Fyll ut skjema" altinn:tasktype="data">
+      <bpmn2:incoming>SequenceFlow_1</bpmn2:incoming>
+      <bpmn2:outgoing>SequenceFlow_2</bpmn2:outgoing>
+    </bpmn2:task>
+    <bpmn2:task id="Task_2" name="Bekreft skjemadata" altinn:tasktype="confirmation">
+      <bpmn2:incoming>SequenceFlow_2</bpmn2:incoming>
+      <bpmn2:outgoing>SequenceFlow_3</bpmn2:outgoing>
+    </bpmn2:task>
+    <bpmn2:task id="Task_3" name="Bekreft skjemadata" altinn:tasktype="feedback">
```

Comment:
The feedback task isn't represented in the policy in any way.. 
but I'm assuming that is because the tests do not cover any actions in this step? 

If we introduce the feedback tasktype we will have to update the process controllers to map feedback to write I'm guessing..

Reply:
Yes, i am thinking write. I will try adding one more test for what agency will use.